scripting/get5.sp

```diff
@@ -549,6 +544,17 @@ public void OnClientAuthorized(int client, const char[] auth) {
   }
 }
 
+public void AdminClientKickCheck(int client) {
+  if (IsPlayer(client)) {
```

Comment:
Added because only Players have to be checked otherwise it will also kick Bots which we require specially GOTV for recording